src/components/victory-axis/helper-methods.js

```diff
@@ -45,13 +44,13 @@ module.exports = {
   getTicks(props, scale) {
     if (props.tickValues) {
       if (Axis.stringTicks(props)) {
-        return range(1, props.tickValues.length + 1);
+        return props.tickValues.map((val, index) => index + 1);
       }
       return props.tickValues;
-    } else if (scale.ticks && typeof scale.ticks === "function") {
+    } else if (scale.ticks && isFunction(scale.ticks)) {
       const ticks = scale.ticks(props.tickCount);
       if (props.crossAxis) {
-        return includes(ticks, 0) ? without(ticks, 0) : ticks;
+        return ticks.indexOf(0) !== -1 ? ticks.filter((val) => val !== 0) : ticks;
```

Comment:
Actually, you're right on that and of course a lib like this can't polyfill so I'm retracting my suggestion. I still think that the lodash version is much cleaner though :)